data-prepper-logstash-configuration/src/main/resources/org/opensearch/dataprepper/logstash/mapping/opensearch.mapping.yaml

```diff
@@ -1,4 +1,5 @@
 pluginName: opensearch
+attributesMapperClass: org.opensearch.dataprepper.logstash.mapping.OpenSearchPluginAttributesMapper
```

Comment:
This same code needs to be applied in `elasticsearch.mapping.yaml` and `amazon_es.mapping.yaml` as well.